.travis.yml

```diff
@@ -29,6 +29,15 @@ install:
 os:
     - linux
 
+branches:
+  only:
+    # This is where pull requests from "bors r+" are built.
+    - staging
+    # This is where pull requests from "bors try" are built.
+    - trying
+    # Uncomment this to enable building pull requests.
+    #- master
```

Comment:
I think this turns off building the PR/master merge which is a test that we require to pass at the moment... so tests haven't even been run on this.

So maybe the right thing here is all of staging/trying/master. (Or maybe we need to turn off that required test and let bors do the enforcement rather than github. I'm not sure.)

Reply:
> So maybe the right thing here is all of staging/trying/master.

I think you're right. In this way, we have the option of using our current workflow or bors.